ansible/roles_ocp_workloads/ocp4_workload_lpe_amq_broker/tasks/remove_workload.yml

```diff
@@ -5,8 +5,8 @@
     state: absent
     definition: "{{ lookup('template', item ) | from_yaml }}"
   loop:
-    - ./../templates/address-queue-create.yaml.j2
```

Comment:
That should never be needed. If your templates are in the templates folder this is implicit. You can just write the template without the folder.